README.md

```diff
@@ -38,6 +38,7 @@ and [good first issues](https://github.com/solid/community-server/issues?q=is%3A
 
 ```shell
 npm ci
+npm run build
```

Comment:
Part of `npm ci`?

Reply:
I may be wrong about this, but nothing in the `npm ci` docs says that it triggers a build (https://docs.npmjs.com/cli/v6/commands/npm-ci). When I cloned a clean probject and followed the instructions, no build was triggered.

It would trigger a build if there were a `"postinstall": "npm run build"` in `package.json`, but there isn't.

Reply:
But there's a `prepare` 😉
You'll see it build when you run it.

Note that it also works for the CI: https://github.com/solid/community-server/blob/v0.3.0/.github/workflows/ci.yml#L34

Reply:
You'd think it would run, but it doesn't when you run `npm ci`. I confirmed it runs the build when you run `npm install`. Perhaps it's because the prepare script will run "Runs on local npm install without any arguments" (https://docs.npmjs.com/cli/v6/using-npm/scripts) and `npm ci` would be considered an install with arguments.

Reply:
```
MacRuben:solid-community-server (feat/build-dist)$ npm --version
6.14.7
MacRuben:solid-community-server (feat/build-dist)$ npm run build

> @solid/community-server@0.3.0 build /Users/ruben/Documents/UGent/Solid/solid-community-server
> npm run build:ts && npm run build:components


> @solid/community-server@0.3.0 build:ts /Users/ruben/Documents/UGent/Solid/solid-community-server
> tsc


> @solid/community-server@0.3.0 build:components /Users/ruben/Documents/UGent/Solid/solid-community-server
> componentsjs-generator -s src -c dist/components -i .componentsignore
```

You can also see it trigger a build on the CI: https://github.com/solid/community-server/runs/1511808350?check_suite_focus=true#step:5:34

Reply:
I don't know what to say. It doesn't trigger locally (node v15.2.1 npm 7.0.10)

Reply:
You're saying it: npm 7.0.10 😉

Reply:
Okay I checked and it does work with 6.14.9, but isn't 7.0.10 after that? It's not like I was using an old version of NPM, right? Also is there anywhere in this codebase that says the version of npm you should be using?

Reply:
6.x is currently latest: https://www.npmjs.com/package/npm?activeTab=versions

7 only ships with Node 15 (unstable)

Reply:
Possibly add a .nvmrc with node 12?

Reply:
It's not really what this pull request is trying to fix, so I created a separate one: https://github.com/solid/community-server/pull/413

Reply:
And this point is now tackled in 6ee56a6d672517d2b3b5fa61a28072d981c0e05a